cis/packaging/rudder-synchronize

```diff
@@ -215,14 +223,25 @@ def import_techniques(path):
         import_technique(path)
 
 GMS = {}
-def get_all_generic_methods():
+TECHNIQUES = {}
```

Comment:
I would replace by a unique variable NCF_CACHE, containing the whole response and check only that variable

and return NCF_CACHE["generic_methods"] or ["techniques"]